src/App.tsx

```
@@ -11,11 +13,20 @@ const App = () => (
+                    <Route index element = { <StationsPage/> }/>
+                </Route>
+                <Route path = "/search">
+                    <Route index element = {
+                        <StationInfoContextProvider>
+                            <SearchPage/>
+                        </StationInfoContextProvider>
+                    }/>
                 </Route>
             </Routes>
             <footer>
+                <hr></hr>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)